src/easylogging++.h

```diff
@@ -93,7 +93,7 @@
 #else
 #  define ELPP_OS_MAC 0
 #endif
-#if (defined(__FreeBSD__) || defined(__FreeBSD_kernel__))
```

Comment:
why remove this?

Reply:
I think this is some typo, definitely unintended.